nuxeo-drive-client/nxdrive/data/ui5/js/ndrive-conflicts.js

```diff
@@ -54,16 +54,27 @@ ConflictsController.prototype.updateErrors = function($scope) {
 	$scope.ignoreds = ignoreds;
 }
 ConflictsController.prototype.updateConflicts = function($scope, $interval) {
+    var lastContributorPending = false;
 	self = this;
 	$scope.conflicts = angular.fromJson(drive.get_conflicts());
 	for (var i=0; i<$scope.conflicts.length; i++) {
+		lastContributorPending = lastContributorPending || $scope.conflicts[i].need_refresh;
 		if ($scope.conflicts[i].last_error == "DUPLICATING") {
 			setTimeout( function() {
 				self.updateConflicts($scope, $interval);
 				$scope.$apply();
 			}, 1000);
 		}
 	}
+	if (lastContributorPending) {
```

Comment:
There are 2 types of "users" updates that relate to conflicts:
1. a new user is missing from the table
2. an existing user's full name has changed
1. is taking care by checking the "last_contributor" property from GetFileSystemItem response if it is in the table; if not, retrieve it. Every doc that could be potentially in conflict, would have its "details" retrieved by this automation API at some point, I think.
2. will be taken care by the daily refresh, as the first/last name of the user don't change that often, and even if they do, the consequence of displaying a full name with a spelling error for example, are minor.
As far as a user being deleted from the system, we can either ignore it or reconcile it during the daily refresh.

I think this address your concerns about a missing user when a conflict is detected.